package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "eslint ."
   },
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=8.13.0"
```

Comment:
Isn't dropping support for an old Node version a breaking change? https://docs.npmjs.com/files/package.json#engines

Reply:
Not sure if anyone's actually running on `node <= 8.13.0`

Reply:
Is 8.13.0 a requirement? Is it the minimum requirement?
Which changes require this?

Reply:
It is currently a requirement. These changes were added prior to this PR. If you run tests on an engine below this version you'll get errors. This is because `doesNotReject()` (https://nodejs.org/docs/latest-v8.x/api/assert.html#assert_assert_doesnotreject_block_error_message) and `rejects()` (https://nodejs.org/docs/latest-v8.x/api/assert.html#assert_assert_rejects_block_error_message) were added in `V8.13.0`.

We are using them here https://github.com/bbc/http-transport-cache/blob/212bb0339ef79586cb77fa052e7e103a997a8bea/test/maxAge.js#L4 and here https://github.com/bbc/http-transport-cache/blob/212bb0339ef79586cb77fa052e7e103a997a8bea/test/staleIfError.js#L4